airflow/ui/src/components/ClearRun/ClearRunTaskAccordion.tsx

```diff
@@ -93,6 +101,34 @@ const ClearRunTasksAccordion = ({ affectedTasks }: Props) => (
         </Box>
       </Accordion.ItemContent>
     </Accordion.Item>
+    <Accordion.Item key="note" value="note">
```

Comment:
Why do you render another accodion around the note? This means another click is needed to see previous note and change it. I assume the field is small enough (can hold <1000 chars anyway, not very long text...) - I would rather propose to render it directly - w/o accordion. Then people are more easily encouraged to leave a note - I think this feature is not used very much because the entry was missing in the past.

Reply:
This is an Item. `Accrodion.Item` those are nested inside the `Accordion.Root`. So basically 1 accordion component, and 2 items inside it. (1 for the note, one for the task instance table).

Reply:
Example here, both can't be opened at the same time if the table is big enough, or everything will overflow and it looks not great.
![Screenshot 2025-01-06 at 17 28 23](https://github.com/user-attachments/assets/c4daa92b-ca95-4516-98a6-471327e41f28)

Reply:
Can we then open the note per default?

Reply:
Good idea, done.